.golangci.yml

```diff
@@ -13,3 +13,4 @@ linters:
     - gosimple
     - unused
     - typecheck
+    - forbidigo
```

Comment:
What does this do?

Reply:
https://github.com/ashanbrown/forbidigo

It's a golanglint check that looks for `fmt.Print` and friends! (It can be configured to look for other regexes too)